pkg/app/app.go

```diff
@@ -56,7 +57,9 @@ func initWindow(width, height int, title string) *glfw.Window {
 		panic(err)
 	}
 	gl.Enable(gl.DEPTH_TEST)
-	gl.DepthFunc(gl.LESS)
+	gl.DepthMask(true)
+	gl.DepthFunc(gl.LEQUAL)
+	gl.DepthRange(0.0, 1.0)
 	gl.Enable(gl.BLEND)
```

Comment:
I've removed the Blend code from Context.Clear(). It was still there because I don't like completely the split we have between the window handling in App and the GL code in Context.
This is anyway a better place